### PR TITLE
Ubuntu Bionic Images

### PR DESCRIPTION
**Pull request by iamjohnnym**
Docker images for Ubuntu 18.04 Bionic Support.  The images reflect closely to the existing images in play.  I was successful in building each of these locally.

socrata/base => socrata/base-bionic
socrata/runit => socrata/runit-bionic
socrata/python => socrata/python-bionic/2.7
socrata/python3 => socrata/python-bionic/3.6
socrata/runit-python => socrata/runit-bionic-python/2.7
                                   => socrata/runit-bionic-python/3.6

### CODE REVIEW

**Comment by znep** [comment]
Seems reasonable at first glance.  Does it make sense to try to DRY things up a bit by referencing things like `clortho-get` in a shared place instead of just copying them into every base image?  Not sure if any of them have differences that have creeped in ...

**Comment by iamjohnnym** [comment]
I considered this when I was looking at the different repo's prior to starting.  It would probably be best to have a single location where common files reside.

1) A separate docker image with `essential-files`, that can be included with something like,
`FROM socrata/build-essentials` 
2) A single repo's where this resides and the ability to do a git clone
3) they reside in `shipyard/scripts` and each base docker dir has a symlink so that they are readily accessible.

thoughts?

File: base-bionic/clortho-get [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+require 'aws-sdk-resources'
+require 'yaml'
+require 'base64'
+
+if ENV['AWS_REGION'].nil?
+  Aws.config[:region] = 'us-west-2'
+  puts "DEBUG: AWS_REGION unset. Defaulting to us-west-2"
+end
+
+if File.file?('/etc/aws/credentials')
```
Comment (iamjohnnym) [inline base-bionic/clortho-get]: This block doe not exist in `runit-bionic/clortho-get`

**Comment by znep** [comment]
Yeah, it is annoying you can't just reference files in other directories.
I think you could make it work but not in a docker hub build.  A separate
image or repo sounds like overkill a bit... symlinks could work assuming
docker isn't silly.   Or we could just not worry for now.

**Comment by iamjohnnym** [comment]
Bleh.  No dice for now.  Leveraging files from a dir above the Dockerfile has been an issue since 2013.  Sad days.

https://github.com/moby/moby/issues/2745 which became https://github.com/moby/moby/issues/37129

**Pull request merged**